src/pages/documentation/faq.md

```diff
@@ -173,6 +173,34 @@ Unresolved class 'FullscreenMessageActivity'
 
 To resolve the build warning, remove FullscreenMessageActivity from your application's manifest file. Campaign Standard SDK no longer requires application to add **FullscreenMessageActivity** to their manifest.
 
+### Why do I see 'java.lang.NoSuchMethodError' after upgrading to the latest version of Mobile SDK for Android?
```

Comment:
since these questions are related to the migration to 2.x, can we group them under an H2 header? 
e.g. `Migrating to Mobile Core 2.x`

Reply:
I like this, actually! Let's do that as well.

Reply:
I would like to have "Android" in the heading. `Migrating to Android Mobile Core 2.x`, Migrating to the latest Mobile SDK for Android`, `Migrating to Android Mobile Core 2.x and supported extensions`

Reply:
Migrating to Mobile Core SDK 2.x for Android and compatible extensions ?